bin/report_runs.py

```diff
@@ -133,6 +140,11 @@ def report_runs(run_ids, noemail=False):
                         round(clean_yield/1000000000, 1), int(sdata['required_yield']/1000000000),
                         round(mean_cov, 1), sdata['required_coverage']
                     )
+                # Checking for other run elements which are still pending
+                for sample_run_id in query_dict(sdata, 'run_elements') or []:
+                    if query_dict(run_data(sample_run_id), 'aggregated.most_recent_proc.status') == 'processing':
```

Comment:
`run_elements` will not match `run_id` and nothing will be found by `query_dict`.
a run element has the format : `<runid>_<lane>_<barcode>`. 
You should parse the run element to extract the run id.

Reply:
Thanks for noting this Tim. I've had a look and a run_element would be '190528_E00373_0401_BH2FHHCCX2_2'. Removing the last number and underscore leaves '190528_E00373_0401_BH2FHHCCX2', which matches a run_id. Is it acceptable if I simply remove the last two characters, up to the underscore?